src/main/java/com/geronimo/model/AuditedEntity.java

```diff
@@ -0,0 +1,25 @@
+package com.geronimo.model;
+
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.MappedSuperclass;
+import javax.persistence.SequenceGenerator;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+
+@Setter
+@ToString
+@MappedSuperclass
+abstract class AuditedEntity {
+
+    @NotNull
+    protected Date date_created;
+
+    @NotNull
+    protected Date last_updated;
```

Comment:
According to Java code convention, all properties that are not constants must be written in came-case.

e.g. lastUpdated, countOfReblogs, countOfLikes, etc.